result.py

```diff
@@ -46,10 +46,16 @@ def calculate_results(self):
                 self.ttft = 1000 * (
                     self.first_token_time - self.start_time
                 )  # Time to first token in ms
-                self.itl = (1000 * (self.end_time - self.first_token_time)) / (
-                    self.output_tokens - 1
-                )  # Inter-token latency in ms. Distinct from TPOT as it excludes the first token time.
-
-            self.tpot = (
-                self.response_time / self.output_tokens
-            )  # Time per output token in ms
+                if self.output_tokens is not None and self.output_tokens > 0:
```

Comment:
this looks quite like
```suggestion
                if self.output_tokens:
```
as `output_tokens` cannot be negative.
Not sure what's better...

(same below)

Reply:
ouuhhh I got the actual error self.output_tokens is 1